### PR TITLE
media-libs/opencolorio: fix bug 916329 914863

### PR DESCRIPTION
**Pull request by negril**
Closes: https://bugs.gentoo.org/914863
Closes: https://bugs.gentoo.org/916329

### CODE REVIEW

File: media-libs/opencolorio/opencolorio-2.2.1.ebuild [hunk 1]
```diff
@@ -125,6 +126,15 @@ src_configure() {
 	cmake_src_configure
 }
 
+src_install()
+{
```
Comment (juippis) [inline media-libs/opencolorio/opencolorio-2.2.1.ebuild:130]: Please don't get into habit of writing this, all ebuilds using this syntax were recently cleared to have a common style.
```suggestion
src_install() {
```

Comment (negril) [reply media-libs/opencolorio/opencolorio-2.2.1.ebuild]: Fixed. Thanks!

**Review by juippis — COMMENTED** [review]
At least docs build now, but
```
 * QA Notice: The ebuild is installing to one or more unexpected paths:
 * 
 *   /usr/share/doc/OpenColorIO
 * 
 * Please fix the ebuild to use correct FHS/Gentoo policy paths.
```

EDIT: readme file seems to install under correct path, but html files not.

**Comment by negril** [comment]
That should be fixed in https://github.com/gentoo/gentoo/pull/33563/commits/d5fb60b7d556f34621ece79eb106f52d78d1d047.

**Pull request closed without merging**